src/main.rs

```diff
@@ -152,6 +152,13 @@ fn main() {
                 .value_name("FILE")
                 .help("Export the timing results to the given file in JSON format."),
         )
+        .arg(
+            Arg::with_name("export-md")
+                .long("export-md")
+                .takes_value(true)
+                .value_name("FILE")
+                .help("Export the timing results to a markdown table in the given file"),
```

Comment:
Could this be `Export the timing results as a Markdown table to the given FILE.`? I have just changed the other two help messages.